kvmagent/kvmagent/test/http/test_ha_lifecycle.py

```
@@ -0,0 +1,109 @@
+
+    @pytest.fixture(autouse=True)
+    def _cleanup(self, http_client):
+        """Remove test fencer rules after each test."""
+        yield
+        try:
+            http_client.post_async('/remove/vm/fencer/rule/from/host', {
+                'allowRules': [{'fencerName': self.FENCER_NAME, 'vmUuids': [self.FAKE_VM_UUID]}],
+                'blockRules': [],
+            })
+        except Exception:
+            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)